### DISCUSSION TITLE
Error loading assets

### DISCUSSION THREAD

**Discussion by tttigo** [mozilla/hubs#3665] [Hubs Cloud Questions] [answered]
Hello,

Every time I open my hubs space, it fails to load all assets.

![image](https://user-images.githubusercontent.com/6611885/103805901-fe4d5580-5032-11eb-96dc-b689b1651f44.png)

Every time I have to refresh the page, and then it loads all of it only on the second try

![image](https://user-images.githubusercontent.com/6611885/103805932-0a391780-5033-11eb-87bf-a5e6e8537782.png)

What can be happening?

**Comment by robinkwilson** [comment, marked as answer]
Are you running a Personal/Enterprise 1 server or Enterprise multiserver HC setup?

**Reply by tttigo** [reply]
personal 1 server

t3.medium

**Reply by robinkwilson** [reply]
This looks like a bug, do you mind taking screenshots of your browser's developer tools > Console tab AND any red failed requests in the Network tab?

**Reply by robinkwilson** [reply]
Marking as answered for now, please "unmark as answered" when you send in more info! Thank you!
